sdk/core/core-lro/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.0.1 (2020-02-28)
 
 - `getOperationState()` now returns `TState`.
 - `TState` of `PollerLike` can be a subset of `TState` of `Poller`,
```

Comment:
Do either of these add functionality, or are they bug fixes? If it's additive then it might be better to make it a minor bump.

Reply:
*Technically* it should be minor, but really it was a bugfix for bad typings we had before, so I think we last argued this down to a patch bump. I don't have strong feelings, though.

Reply:
That's fine by me, just wanted to make sure it was thought about.

Reply:
Also I think we're getting to a point where 'breaking change' means "makes JS users have to touch code" rather than simply offering better/refined typings to TS customers.

Reply:
Thank you both!